Git.hub/Client.cs

```diff
@@ -148,6 +148,20 @@ public User getCurrentUser()
             return user.Data;
         }
 
+        public User getUser(string username)
+        {
+            if (username == null)
+                throw new ArgumentNullException(nameof(username));
+
+            if (username.Length == 0)
+                throw new ArgumentException("Empty username", nameof(username));
```

Comment:
Combine both checks into one

```
if (string.IsNullOrWhiteSpace(userName)) 
{
    throw new ArgumentException("Empty username", nameof(userName));
}
```

Reply:
I made separate checks because I thought it would be better to throw `ArgumentNullException` for a null argument... I don't mind changing it if you prefer a single check with a less appropriate exception.

Reply:
Less code the better :)
IMO two checks don't really provide any additional value in this case.